test/e2e/kubevirt.go

```diff
@@ -1372,9 +1424,18 @@ chpasswd: { expire: False }`
 				Should(ConsistOf(expectedAddresesAtGuest), step)
 
 			step = by(vmi.Name, fmt.Sprintf("Check east/west traffic before %s %s", td.resource.description, td.test.description))
-
 			checkEastWestIperfTraffic(vmi, testPodsIPs, step)
 
+			nodes, err := e2enode.GetBoundedReadySchedulableNodes(context.TODO(), fr.ClientSet, 1)
+			Expect(err).NotTo(HaveOccurred())
+
+			nodeIPs := e2enode.CollectAddresses(nodes, v1.NodeInternalIP)
+
+			if td.role == "primary" {
+				step = by(vmi.Name, fmt.Sprintf("Check north/south traffic before %s %s", td.resource.description, td.test.description))
+				checkNorthSouthIperfTraffic(externalContainerName, nodeIPs, svc.Spec.Ports[0].NodePort, step)
```

Comment:
Can you help me understand what is the `checkNorthSouthIperfTraffic` actually checking? I would have thought this has to be ran from the VM but I am probably missing something.

Reply:
This comments should be done at the Dependency PR -> https://github.com/ovn-kubernetes/ovn-kubernetes/pull/4834